core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -214,11 +216,13 @@ export class AudioBarComponent {
     if (this.autogeneratedAudioPlayerService.isPlaying()) {
       this.autogeneratedAudioPlayerService.cancel();
     } else {
+      let speechSynthesisLanguageCode =
+        this.audioTranslationLanguageService.getSpeechSynthesisLanguageCode();
       this.autogeneratedAudioPlayerService.play(
         this.audioTranslationManagerService
           .getCurrentHtmlForAutogeneratedAudio(),
-        this.audioTranslationLanguageService
-          .getSpeechSynthesisLanguageCode(),
+        speechSynthesisLanguageCode === null ?
```

Comment:
Ditto about string value of null

Reply:
I did this because `getSpeechSynthesisLanguageCode()` in [audio-translation-language.service.ts](https://github.com/oppia/oppia/blob/develop/core/templates/pages/exploration-player-page/services/audio-translation-language.service.ts) can return null while `this.autogeneratedAudioPlayerService.play()` does not allowed `null` type parameters. Would you like me to initalize  `speechSynthesisLanguageCode` to something else (such as. `'en'`) if `getSpeechSynthesisLanguageCode()` returns null?

Reply:
Ah, yes defaulting to `en` makes sense in this case, I think. Thanks for checking!